helm_deploy/cla-frontend/values.yaml

```diff
@@ -2,7 +2,7 @@
 # This is a YAML-formatted file.
 # Declare variables to be passed into your templates.
 
-replicaCount: 1
+replicaCount: 4
```

Comment:
I think this can stay as 1, as it's used for all environments but production overwrites it in the `helm_deploy/cla-frontend/values-production.yaml` file

Reply:
ok